src/util/valueSetHelper.js

```diff
@@ -0,0 +1,41 @@
+function getHierarchicalCodes(valueSet) {
```

Comment:
My rule of thumb is typically that if a function isn't exported, then it doesn't *need* to have a jsdoc style comment on it (sometimes still nice to have). I actually am not sure this function needs to be exported since it's only used within this file (your call if you think it will be useful outside of this file). But would you mind at least adding jsdoc style comments on all exported functions?